modules/exploits/linux/local/nested_namespace_idmap_limit_priv_esc.rb

```diff
@@ -124,7 +124,7 @@ def upload_and_compile(path, data)
     end
 
     register_file_for_cleanup path
-    chmod path, 0755
+    chmod path, 0o755
```

Comment:
Typo?

```suggestion
    chmod path, 0755
```

Reply:
https://github.com/dubledorm/mcs_control/blob/master/.rubocop.yml#L62-L64

looks like this is rubocop...

Reply:
The `0o` is just a prefix for signifying that the number is in octal format, similar to `0x` or `0b` - https://docs.ruby-lang.org/en/2.0.0/syntax/literals_rdoc.html#label-Numbers